rxjava3/build.gradle

```diff
@@ -2,7 +2,9 @@ dependencies {
     annotationProcessor "io.micronaut:micronaut-inject-java:$micronautVersion"
     annotationProcessor "io.micronaut:micronaut-graal:$micronautVersion"
     api "io.micronaut:micronaut-runtime:$micronautVersion"
-    api "io.reactivex.rxjava3:rxjava:$micronautRxJava3Version"
+    api "io.reactivex.rxjava3:rxjava:$rxJava3Version"
+
+    compileOnly "io.reactivex.rxjava2:rxjava:$rxJava2Version"
```

Comment:
Just curious, why do we need rxjava2?

Reply:
Due to `RxJava2ToRxJava3ConveterRegistrar`. It is a `compileOnly` dependency.